UnhollowerBaseLib/ClassInjector.cs

```diff
@@ -147,16 +155,21 @@ public static void RegisterTypeInIl2Cpp(Type type, bool logSuccess)
             methodPointerArray[0] = ConvertStaticMethod(FinalizeDelegate, "Finalize", classPointer);
             var finalizeMethod = UnityVersionHandler.Wrap(methodPointerArray[0]);
             methodPointerArray[1] = ConvertStaticMethod(CreateEmptyCtor(type), ".ctor", classPointer);
+            Dictionary<(string, bool), int> infos = new Dictionary<(string, bool), int>(eligibleMethods.Length);
```

Comment:
I have a slight suspicion that this dictionary will break really fast on interfaces that have method overloads - might want to *at least* add parameter count to keys here, but ideally keys should be full method signatures.

Reply:
I'm currently oposed to adding a full method signature as e.g. pinned/ref parameters aren't supported, but a user could still try to overload it.